cleanlab/filter.py

```diff
@@ -200,6 +201,11 @@ class 0, 1, ..., K-1.
       Obtain the *indices* of label issues in your dataset by setting
       `return_indices_ranked_by`.
     """
+    if not rank_by_kwargs:
+        rank_by_kwargs = {}
+
+    if not pred_probs:
+        pred_probs = np.ndarray([])
```

Comment:
`pred_probs` should remain as required positional arguments.
It should be validated by `assert_valid_inputs` early in the function.

```suggestion
```

Reply:
So the doc string is wrong at that spot?
> pred_probs : np.ndarray, optional

Reply:
I'll have to think about that.